.github/workflows/tests.yml

```
@@ -24,7 +24,7 @@ jobs:
     - name: Set up Ruby
       uses: actions/setup-ruby@v1
       with:
-        version: '>=2.3'
+        ruby-version: '>=2.3'
```

Should we also bump this to 2.6 given the recent upgrade of Homebrew/brew?

```suggestion
        ruby-version: '>=2.6'
```

---

Whoops, just missed this! Yes, this seems like a good idea to do across all the repos in the org?

---

Cool, I’ll do this tomorrow unless anyone gets there first.